R/util_assert.R

```diff
@@ -0,0 +1,25 @@
+assert_scalar <- function(x, name = deparse(substitute(x)), arg = name,
```

Comment:
After your change you no longer need these :) but they're bound to come in useful lter